pkgs/tools/admin/zbctl/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib, stdenvNoCC, fetchurl, coreutils }:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "zbctl";
+  version = "8.0.6";
+
+  dontUnpack = true;
+  dontConfigure = true;
+  dontBuild = true;
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out/bin
+    cp $src $out/bin/zbctl
+    chmod +x $out/bin/zbctl
+    runHook postInstall
+  '';
+
+  src = if stdenvNoCC.hostPlatform.system == "x86_64-darwin" then fetchurl {
```

Comment:
The standard order of the attributes is something like this:

```
stdenvNoCC.mkDerivation rec {
  pname
  version

  src

  (variables influencing the build such as dontUnpack)

  (phases such as installPhase)

  meta
}
```
You should use that here as well.